WebisD/http-api-without-lib
# Feat create get method and fix post not rendering

### **Why**
Created a functional GET method so user can request resources from the server and also fixed a bug in the POST method so the user can add new data to the database.json

### **What was done**
- Converted response dictionary to Response class
- Fixed POST headers so it can render a page when posting
- Fixed post.html, so it can send the correct form data
- Added a request parser for the form data

### **How to Test**
- Run the `main.py` file on your terminal of choice
- Acess the localhost:port/post and fill the data:

![image](https://user-images.githubusercontent.com/63318342/115624264-c2cc5380-a2d0-11eb-90ed-ee209078b338.png)

- Check if the response html is the same as the following:

![image](https://user-images.githubusercontent.com/63318342/115624342-df688b80-a2d0-11eb-8c37-e4b0b9a959ff.png)
